### PR TITLE
Api hit

### PR DESCRIPTION
**Pull request by kazim110**
Add Api `Get `and `Fetch`

### CODE REVIEW

File: src/modules/ui.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import GetScore from './getFromAPi.js';
```
Comment (muneebulrehman) [inline src/modules/ui.js]: I believe you have misspelled the path for this import and it is causing all sorts of errors as you can see in the screenshot below. ⬇️ Kindly fix this error. Except for this everything looks good.
image

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @kazim110 , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Good code structure

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly use the `async` and `await` instead of `promises` where ever you have requested/posted data from API. It is a part of the requirement.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @kazim110, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Code structure ✔
- Made requested changes ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/getFromAPI.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import URL from './url.js';
+
+const getScore = async (gameId = 'gTYDGIDW8fMHrK3hEdRJ') => fetch(`${URL}/games/${gameId}/scores`)
```
Comment (muneebulrehman) [inline src/modules/getFromAPI.js:3]: test

**Pull request merged**